packages/fast-usdc/src/types.ts

```diff
@@ -55,6 +55,11 @@ export interface TransactionRecord extends CopyRecord {
   status: TxStatus;
 }
 
+export interface ContractRecord extends CopyRecord {
```

Comment:
the name... does it refer to "the record you'll get from vstorage if you query the contract's node"?

docstring, please?